airflow/providers/cncf/kubernetes/hooks/kubernetes.py

```diff
@@ -502,6 +503,30 @@ def create_job(
             raise e
         return resp
 
+    def get_job(self, job_name: str, namespace: str) -> V1Job:
```

Comment:
I'm wondering, if this is required for the Google provider, why don't we add these methods to the Google providers?

Reply:
For this purpose i found it very convenient to use kubernetes api, and since we already had a dedicated hook for Job in kubernetes, i have only extended its functionality